docs/run_test/runbook.rst

```diff
@@ -419,8 +419,9 @@ type: list of Transformer, default is empty
 type
 ^^^^
 
-type: str, required, the type of transformer. See
-`transformers <../../lisa/transformers>`__ for all transformers.
+type: str, required, the type of transformer. See `transformers
+<https://github.com/microsoft/lisa/tree/main/lisa/transformers>`__ for all
+transformers.
```

Comment:
Why not use Sphinx labels?

Reply:
You mean like ``.. some link: xxx.com``? I only replaced the relative part with external one to GitHub since they are all code files in Git.

Reply:
I mean using sphinx.ext.autosectionlabel and just getting away with a ``:ref:`Transformer```, instead of all that gunk (that may break an need maintenance, in any restructuring effort).

Reply:
It works in internal links (between html pages) but does not work on external links, so I'm afraid we have to keep these external links to codes on Git.

Reply:
How is runbook.rst linking to transformers external? I'm lost here...

Reply:
In runbook.rst there's mainly reference to commands, codes etc. The "transformers" here refers to the folder which holds source codes of all transformers, and since we do not have codes on rtd, it's an external link to Git.

Reply:
I see. Time to document the classes, then (one more reason :))